docs/usage/customization-guide.md

```diff
@@ -185,6 +185,37 @@ to specify taints in the NodeFeatureRule object.
 > not tolerate the taint are evicted immediately from the node including the
 > nfd-worker pod.
 
+### kubectl plugin
+
+The `kubectl` plugin `kubectl nfd` can be used to validate/dryrun and test
+NodeFeatureRule objects. It can be installed with the following command:
+
+```bash
+make build-kubectl-nfd
+KUBECTL_PATH=/usr/local/bin/
+mv ./bin/kubectl-nfd ${KUBECTL_PATH}
+```
```

Comment:
Add instructions to do git clone (and other steps)

Just a note that we can probably start with a "developer preview" and go with this but going forward we want to have this as one of the release assets